frontend/test/pytest/test_callback.py

```diff
@@ -918,6 +918,66 @@ def expected(x):
     # Array(0.4565774, dtype=float64)
 
 
+@pytest.mark.parametrize("dtype", [jnp.float32, jnp.float64])
```

Comment:
@mehrdad2m I meant on line 888. This test is the same except for the dtype.

Reply:
@erick-xanadu Oh yes you are right. I fixed it by removing the redundant test and parametrize the original test case.